config/method_config.go

```diff
@@ -70,9 +76,34 @@ func initProviderMethodConfig(sc *ServiceConfig) error {
 }
 
 // check set default value and verify
-func (mc *MethodConfig) check() error {
-	if err := defaults.Set(mc); err != nil {
+func (m *MethodConfig) check() error {
+	qualifieldMethodName := m.InterfaceName + "#" + m.Name
+	if m.TpsLimitStrategy != "" {
+		_ = extension.GetTpsLimitStrategyCreator(m.TpsLimitStrategy)
```

Comment:
why we call this? you don't use the return value.

Reply:
I prefer that you add one more function to check if the creator exist. And then you log error here.

Reply:
call this to check if we enable the corresponding strategy, maybe adding one more function is a better way to detect this, I'll fix it.